<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(99, 69, 44);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#1a5fb4;&quot;&gt;Fiche de Renseignements&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Matricule:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Prénom:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Nom:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Date Naissance:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Email:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Genre:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Section:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Moyenne:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#613583;&quot;&gt;Option(s):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_ajouter">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>580</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(99, 69, 44);</string>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_annuler">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>580</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(99, 69, 44);</string>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="matricule">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="prenom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nom">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="daten">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_masculin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Masculin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="r_feminin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Féminin</string>
    </property>
   </widget>
   <widget class="QComboBox" name="section">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>430</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Selectionner Votre Section</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sciences Informatiques</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sciences Expérimentales</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mathématiques</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Techniques</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lettres</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Economie Gestion</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sport</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="c_dessin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Dessin</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="c_italien">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>530</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Italien</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="c_musique">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>530</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Musique</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="moyenne">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_aff_tous_elev">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher tous les élèves</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_classement">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>40</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Classement</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="t_toute_eleves">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>601</width>
      <height>231</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Matricule</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nom &amp; Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Genre</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Section</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Moyenne</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Option(s)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="b_nompre_admis">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>340</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom et Prénom Des Admis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_percentage">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(99, 69, 44);</string>
    </property>
    <property name="text">
     <string>Percentage</string>
    </property>
   </widget>
   <widget class="QListWidget" name="l_eleve_admis">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>380</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_quitter">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>600</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>420</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#26a269;&quot;&gt;Admis:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>470</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#e5a50a;&quot;&gt;Ajournés:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>520</y>
      <width>66</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#a51d2d;&quot;&gt;Refusés:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="i_admis">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="i_ajourne">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>470</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="i_refuse">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevoir_Revision">
    <property name="title">
     <string>Devoir Revision</string>
    </property>
   </widget>
   <addaction name="menuDevoir_Revision"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
